From: David Herrmann <dh.herrmann@gmail.com>
Date: Sun, 3 Nov 2013 16:17:14 +0100
Subject: rules: simply 60-drm.rules

We don't need any GOTO, if we merge all matches into a single line.
---
 rules/60-drm.rules | 7 +------
 1 file changed, 1 insertion(+), 6 deletions(-)

diff --git a/rules/60-drm.rules b/rules/60-drm.rules
index de038c1..1ed3e44 100644
--- a/rules/60-drm.rules
+++ b/rules/60-drm.rules
@@ -1,8 +1,3 @@
 # do not edit this file, it will be overwritten on update
 
-ACTION=="remove", GOTO="drm_end"
-SUBSYSTEM!="drm", GOTO="drm_end"
-
-SUBSYSTEMS=="pci|usb|platform", IMPORT{builtin}="path_id"
-
-LABEL="drm_end"
+ACTION!="remove", SUBSYSTEM=="drm", SUBSYSTEMS=="pci|usb|platform", IMPORT{builtin}="path_id"
